inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -46,6 +48,7 @@
 
 
 @with_debug_logging
+@backoff.on_exception(backoff.expo, Exception, max_tries=7)
```

Comment:
Setting this to 7 to reach almost one minute (legacy sometimes throttle clients for 10 seconds...)